ocfweb/docs/docs/services/lab/printing.md

```diff
@@ -6,8 +6,8 @@ start of each semester.
 
 Printing is currently limited to:
 
- * 100 pages per semester
- * 10 pages per day on weekdays
+ * 200 pages per semester
+ * 20 pages per day on weekdays
  * 20 pages per day on weekends
```

Comment:
We could just merge these both to `20 pages per day`

Reply:
Done!